### DISCUSSION TITLE
iOS Build fails after creating multiple schemes

### DISCUSSION THREAD

**Discussion by mostafaAbdelazim** [codemagic-ci-cd/codemagic-docs#2304] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Try to build a multi scheme/flavor app

### Expected results

ios build would pass successfully for all flavors

### Actual results

After starting the build, it fails with this message, i can't tell for sure what's the reason, I can build ipa locally without any issues
<img width="767" alt="Screenshot 2024-01-13 at 6 31 12 AM" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/43379502/3f4614e8-052a-4816-86b8-8705b3a2d923">

**Comment by UsamaLaeeqTech** [comment]
Hi @mostafaAbdelazim I'm also facing same issue If you have found any solution can you please share it.

**Reply by himesh-cm** [reply]
Hi @UsamaLaeeqTech, please open a different thread with information and build ID. You can find it under your build details page's 'ID' section. ﻿Navigate to the 'Builds' section on the left-hand menu, select the build you want to view, and click the copy icon.

**Comment by himesh-cm** [comment]
Hi @mostafaAbdelazim, please share the build id, so I can further investigate the issue. You can find it under your build details page's 'ID' section. ﻿Navigate to the 'Builds' section on the left-hand menu, select the build you want to view, and click the copy icon.

**Reply by mostafaAbdelazim** [reply]
Thank you for your response, here's my build id
65a210be54cf0707a0b01fe7

**Reply by mostafaAbdelazim** [reply]
I've tried some solutions in the pre build script phase as you can see, but without any luck

**Reply by himesh-cm** [reply]
Hi @mostafaAbdelazim, are you able to reproduce this locally?
The issue looks specific to this GitHub issue - https://github.com/AngeloAvv/flutter_flavorizr/issues/17
What platform version do you have in your `Podfile`?

**Reply by mostafaAbdelazim** [reply, marked as answer]
I had to include this line in my flutter/{schemeName}Release.xcconfig file
`#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.release-{schemeName}.xcconfig"`
